AppCenter/AppCenter/Internals/HttpClient/MSHttpClient.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSHttpClientProtocol.h"
```

Comment:
Doesn't look correct order.

Reply:
Agreed but for some reason this is what the formatter script is changing it to.